.github/workflows/aks-auto-start.yaml

```diff
@@ -18,7 +18,14 @@ jobs:
           client-id: 2b6fa9d7-7dba-4600-a58a-5e25554997aa # DTS AKS Auto-Shutdown
           tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
           allow-no-subscriptions: true
+      - uses: azure/CLI@v1
+        name: Fetch slack webhook secret
+        with:
+          inlineScript: |
+            az account set -s bf308a5c-0624-4334-8ff8-8dca9fd43783
+            echo "::set-output name=SLACK_WEBHOOK::$(
+                    az keyvault secret show   -n registry-slack-webhook --vault-name infra-vault-sandbox | jq -r .value
+                  )"
+        id: slack
```

Comment:
please use the built-in task

https://github.com/hmcts/dynatrace-availability-dashboards/blob/8338de92ffffb6d2dd92de84003e3b95dd11b3f5/azure-pipelines.yaml#L129-L135